### PR TITLE
Update flags references

### PR DESCRIPTION
**Pull request by DoryanR**
Update flags via [clayton's tool](https://github.com/cl8n/osu-wiki-bin)

### CODE REVIEW

**Review by TPGPL — COMMENTED** [review]
the changes to ASC articles are wrong (*probably should add tooltips to the examples though*)

**Comment by DoryanR** [comment]
I'm aware of it, I'm fixing this problem (already fixed before, but it was not modified on the right commit in #5999)

File: wiki/Article_styling_criteria/Formatting/en.md [hunk 1]
```diff
@@ -791,3 +790,5 @@ These must have an empty line before and after the markup. Thematic breaks must
 ```markdown
 ---
 ```
+
+[flag_AU]: /wiki/shared/flag/AU.gif "Australia"
```
Comment (Bazzadwarf) [inline wiki/Article_styling_criteria/Formatting/en.md:794]: This doesn't need to be changed, revert this change

File: wiki/Article_styling_criteria/Formatting/fr.md [hunk 2]
```diff
@@ -780,3 +779,5 @@ Elles doivent être précédées et suivies d'une ligne vide. Les ruptures thém
 ```markdown
 ---
 ```
+
+[flag_AU]: /wiki/shared/flag/AU.gif "Australie"
```
Comment (Bazzadwarf) [inline wiki/Article_styling_criteria/Formatting/fr.md:783]: This doesn't need to be changed, revert this change

**Comment by DoryanR** [comment]
What does this PR need to move forward?

**Comment by TicClick** [comment]
nothing really I guess, I'll take a look sooner or later

**Review by TicClick — APPROVED** [review]

File: wiki/osu!stream/Song_List/en.md [hunk 3]
```diff
@@ -274,13 +274,13 @@ See also:
...
+[flag_GB]: /wiki/shared/flag/GB.gif "United Kingdom"
 [flag_KR]: /wiki/shared/flag/KR.gif "South Korea"
-[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
-[flag_RU]: /wiki/shared/flag/RU.gif "Russian Federation"
 [flag_MX]: /wiki/shared/flag/MX.gif "Mexico"
 [flag_NZ]: /wiki/shared/flag/NZ.gif "New Zealand"
+[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
+[flag_RU]: /wiki/shared/flag/RU.gif "Russian Federation"
 [flag_SE]: /wiki/shared/flag/SE.gif "Sweden"
-[flag_TW]: /wiki/shared/flag/TW.gif "Alace"
```
Comment (TicClick) [inline wiki/osu!stream/Song_List/en.md:285]: Alace is no longer a country..

**Review by TicClick — APPROVED** [review]
